.github/workflows/sdk-test.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       - name: Test
         env:
           PRIVATE_KEY: ${{ secrets.TESTING_PRIVATE_KEY }}
+          PASSKEY_PRIVATE_KEY: ${{ secrets.TESTING_PASSKEY_PRIVATE_KEY }}
```

Comment:
This variable still needs to be added the secrets for the repo (it seems I don't have the permission to do that).